tests/benchmark/bloatnet/test_bloatnet.py

```diff
@@ -0,0 +1,246 @@
+"""
+abstract: BloatNet bench cases extracted from https://hackmd.io/9icZeLN7R0Sk5mIjKlZAHQ.
+
+   The idea of all these tests is to stress client implementations to find out where the limits of
+   processing are focusing specifically on state-related operations.
+"""
+
+import pytest
+
+from ethereum_test_forks import Fork
+from ethereum_test_tools import (
+    Account,
+    Address,
+    Alloc,
+    Block,
+    BlockchainTestFiller,
+    Transaction,
+    While,
+)
+from ethereum_test_tools.vm.opcode import Opcodes as Op
```

Comment:
There are some folder restructuring in EEST, please rebase first and the path could be updated as follows to avoid conflict.

```suggestion
from ethereum_test_vm.opcodes import Opcodes as Op
```